<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Linkapp联创认证客户端</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/background.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>180</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 75 9pt &quot;宋体&quot;;lineedit-password-character: 9679;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="rememberPassword">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>NoAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="link">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/link.png</normaloff>:/images/link.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>300</width>
      <height>240</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>常规</string>
     </attribute>
     <widget class="QCheckBox" name="autoHide">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>连接成功后自动隐藏</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="autoLink">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>程序启动后自动连接</string>
      </property>
     </widget>
     <widget class="QComboBox" name="interfaceBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="detectNetLink">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>135</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>重新检测网络连接</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>使用网络连接：（目前仅支持有线连接）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>网络参数</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>802.1x认证</string>
      </property>
      <widget class="QSpinBox" name="maxInterval">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>该值过小，容易中途掉线，该值过大，与服务器失去连接时会很久才能发现，不建议修改该值</string>
       </property>
       <property name="minimum">
        <number>30</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="retryTimes">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>45</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>“寻找服务器”时发生超时后重复尝试的次数，过小会经常找不到服务器</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>最大认证间隔（秒）</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="waitRespond">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>该值过小，会容易发生找不到服务器的情况</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>等待服务器响应（秒）：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>45</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>重试次数 ：</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QPushButton" name="about">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>关于程式……</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/aboutProgram.png</normaloff>:/images/aboutProgram.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="aboutAuthor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>关于作者……</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/aboutAuthor.png</normaloff>:/images/aboutAuthor.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
    <property name="buddy">
     <cstring>password</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>150</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
    <property name="buddy">
     <cstring>account</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="account">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>150</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="linkType">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Raavi</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>@internet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>@edu</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="disconnect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>255</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>应用&amp;A</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/opinion-okay.png</normaloff>:/images/opinion-okay.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>255</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>取消&amp;C</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/opinion-no.png</normaloff>:/images/opinion-no.png</iconset>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>20</width>
      <height>301</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>215</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>联创802.1x 认证</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电信PPPOE 拨号</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>220</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>认证：</string>
    </property>
    <property name="buddy">
     <cstring>comboBox</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="whatsthis">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>215</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>这是什么？</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>link</tabstop>
  <tabstop>settings</tabstop>
  <tabstop>exit</tabstop>
  <tabstop>disconnect</tabstop>
  <tabstop>account</tabstop>
  <tabstop>linkType</tabstop>
  <tabstop>password</tabstop>
  <tabstop>rememberPassword</tabstop>
  <tabstop>autoHide</tabstop>
  <tabstop>autoLink</tabstop>
  <tabstop>waitRespond</tabstop>
  <tabstop>retryTimes</tabstop>
  <tabstop>maxInterval</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>apply</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
